src/InputValidationScript.tsx

```diff
@@ -0,0 +1,34 @@
+import e from "express";
+import React from "react";
+import { isRegExp } from "util/types";
+
+function validateForm(email: string, password: string)
+  {    
+  validateEmail(email);
+  validatePassword(password);
+}
+
+function validateEmail(email: string) {
+  var emailRegex = /^[\w-\.]+@([\w-]+\.)+[\w-]{2,4}$/
```

Comment:
Com a existencia de `const` e `let`, é preferivel evitar o uso de `var`. Sugiro o estudo de diferença de escopo no javascript. Exemlpo: https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/